pkgs/by-name/jf/jfbview/package.nix

```
@@ -0,0 +1,68 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  versionCheckHook,
+  nix-update-script,
+  freetype,
+  harfbuzz,
+  imlib2,
+  libjpeg,
+  ncurses,
+  openjpeg,
+  zlib,
+  xorg,
+  cmake,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "jfbview";
+  version = "0.6.0";
+
+  src = fetchFromGitHub {
+    owner = "jichu4n";
+    repo = "jfbview";
+    tag = finalAttrs.version;
+    hash = "sha256-ASgKXk5iVqKsBc1uzakHs5ojUb6AptGm9LxNyFcDngc=";
+    fetchSubmodules = true;
```

Nit:
```suggestion
    fetchSubmodules = true;
    hash = "sha256-ASgKXk5iVqKsBc1uzakHs5ojUb6AptGm9LxNyFcDngc=";
```
As `fetchSubmodules` impacts the hash value, I would recommend putting it above. It makes it more obvious.